packages/core/src/index.ts

```diff
@@ -85,26 +95,45 @@ export const create = (config: Config): VoiceCompass => {
     },
   });
 
-  let stepId: string | null = null;
+  let _stepId: string | null = null;
```

Comment:
I'm using stepId when destructuring the StepData

Reply:
For readability I would prefer if we didn't destructure instead. It is clearer to me that somewhere down the line there are `stepData.stepId`, which is different from this top-level `stepId`, as opposed to seeing `stepId` and `_stepId`.